tests/robot-tests/tests/admin_and_public_2/bau/publish_release_and_amend.robot

```diff
@@ -559,6 +582,15 @@ Update second accordion section text for amendment
     user adds content to autosaving accordion section text block    Test text    1    Updated test text!
     ...    css:#releaseMainContent
 
+Update embedded dashboard title
+    user updates embedded dashboard in editable accordion section
+    ...    Test embedded dashboard section
+    ...    Test embedded dashboard title updated
+    ...    https://department-for-education.shinyapps.io/dfe-shiny-template/
+    ...    css:#releaseMainContent
+
+    user closes accordion section    Test embedded dashboard section    css:#releaseMainContent
```

Comment:
could potentially use id:releaseMainContent here

Reply:
I've changed all instances across the UI tests.